core/src/main/java/org/apache/carbondata/core/datamap/TableDataMap.java

```diff
@@ -138,8 +138,12 @@ public CarbonTable getTable() {
       }
     }
     int numOfThreadsForPruning = CarbonProperties.getNumOfThreadsForPruning();
+    int carbonDriverPruningMultiThreadEnableFilesCount =
```

Comment:
Need to add validation for carbon property, if someone configures negative value. Need to use the default value

Reply:
resolved